storage/innobase/xtrabackup/doc/source/installation.rst

```diff
@@ -71,13 +73,13 @@ Binary tarballs are available for download and installation.
      - percona-xtrabackup-<version number>-Linux.x86_64.glibc2.12-minimal.tar.gz
      - Contains binaries, and libraries but does not include test files, or debug symbols
 
-Select the |Percona XtraBackup| 8.0 version number and the type of tarball for your installation. Both binary tarballs support all distributions.
+Selecting a different software, such as Ubuntu 20.04 (Focal Fossa), provides a tarball for that operating system. You can download the packages together or separately. 
 
-Fetch and extract the correct binary tarball. For example, the following downloads the full tarball:
+The following link is an example of downloading the full tarball for Linux/Generic:
 
 .. code-block:: bash
 
-  $ wget https://www.percona.com/downloads/Percona-XtraBackup-8.0/LATEST/percona-xtrabackup-8.0.13-Linux.x86_64.glibc2.12.tar.gz
+  $ wget https://downloads.percona.com/downloads/Percona-XtraBackup-LATEST/Percona-XtraBackup-8.0.23-16/binary/tarball/percona-xtrabackup-8.0.23-16-Linux-x86_64.glibc2.12.tar.gz
 
```

Comment:
2.12 is for CentOS6, so 2.17 might be a better example for other distros (or you can add both)

Reply:
Fixed to 2.17